lib/icingadb/icingadb-objects.cpp

```diff
@@ -631,18 +617,21 @@ void IcingaDB::InsertObjectDependencies(const ConfigObject::Ptr& object, const S
 				if (runtimeUpdate || m_DumpedGlobals.CustomVar.IsNew(kv.first)) {
 					allCvs.emplace_back(kv.first);
 					allCvs.emplace_back(JsonEncode(kv.second));
-				}
 
-				if (runtimeUpdate) {
-					publishes["icinga:config:update:customvar"].emplace_back(kv.first);
+					if (runtimeUpdate) {
+						Dictionary::Ptr data = kv.second;
```

Comment:
I was unable to do an inline cast to Dictionary within the function call.

Reply:
Have you tried making the parameter a `const Ptr&`?

Reply:
But then I would have to cast back to a Dictionary to access and modify the underlaying data, right? It seems way easier to just have this extra line here.

Reply:
Try it out. If an extra cast is required, the compiler will fail.

Reply:
I really don't think that it's a good idea to use a more generic type, just because we want to avoid one extra line. What's the problem with this line?

Reply:
Which "more generic type"? Have you tried to change the function parameter's type `Dictionary::Ptr&` -> `const Dictionary::Ptr&` or not?

Reply:
Ohh, I totally misunderstood what you meant. That seems to work 👍